plugins/Network/plugin.py

```diff
@@ -165,7 +165,24 @@ def cmdall(self, irc, msg, args, commandAndArgs):
         for ircd in ircs:
             self.Proxy(ircd, msg, commandAndArgs)
     cmdall = wrap(cmdall, ['admin', many('something')])
-    
+
+    def asrvchanscmd(self, irc, msg, args, commandAndArgs):
```

Comment:
Is this command really needed? You should be able to run `asrvchanscmd` through `cmdall`.